draft-looker-cfrg-bbs-signatures.md

```diff
@@ -697,45 +697,52 @@ There are multiple ways in which messages can be mapped to their respective scal
 
 ### MapMessageToScalarAsHash
 
-This operation takes an input message and maps it to a scalar value via a cryptographic hash function for the given curve.
+This operation takes an input message and maps it to a scalar value via a cryptographic hash function for the given curve. The operation takes also as an optional input a domain separation tag (dst). If a dst is not supplied, its value MUST default to the octet string returned from utf8(ciphersuite\_id || "MAP\_MSG\_TO\_SCALAR\_AS\_HASH\_"), where ciphersuite\_id is the ASCII string representing the unique ID of the ciphersuite.
 
 ```
 result = MapMessageToScalarAsHash(msg, dst)
 
 Inputs:
 
 - msg (REQUIRED), octet string.
-- dst (REQUIRED), octet string. Domain separation tag; note this is not
-                  defined as a function argument as per
-                  [@!I-D.irtf-cfrg-hash-to-curve] but as a parameter.
+- dst (OPTIONAL), an octet string representing an ASCII encoded domain
+                  separation tag. If not supplied, it defaults to the
+                  octet string given by
+                  utf8(ciphersuite_id || "MAP_MSG_TO_SCALAR_AS_HASH_"),
+                  where ciphersuite_id is defined by the ciphersuite.
```

Comment:
I don't think the DST needs to be defined as 'ASCII encoded' here, as technically any octet string is accepted.

Reply:
Done. Thanks!